pom.xml

```diff
@@ -99,6 +107,18 @@
 	</build>
 	<profiles>
 		<profile>
+			<id>weld.test.3plus</id>
+			<activation>
+				<property>
+					<name>weld.test.3plus</name>
```

Comment:
Shouldn't here weld.test.version get change to 3.0.0.FINAL or whatever the best default might be, so only the profile needs to be set when calling maven?